ui/src/app/app.component.html

```diff
@@ -26,7 +26,7 @@
   <header class="header-6" *ngIf="isAuthenticated">
     <div class="branding">
       <a routerLink="{{routes.WORKFLOWS}}" class="nav-link">
-        <clr-icon shape="dashboard"></clr-icon>
+        <img class="clr-icon" src="assets/header_logo.svg">
```

Comment:
Please add the alt text, as reported by Sonar

Reply:
👍🏻